app/models/scraper.rb

```diff
@@ -44,10 +44,15 @@ def search_data
     {
       full_name: full_name,
       description: description,
-      scraped_domain_names: scraped_domain_names
+      scraped_domain_names: scraped_domain_names,
+      has_data?: has_data?
```

Comment:
Searchkick reindexes this by default, like the scraped domains, and I've tested it reindexes when a scraper that has no data gets run and then has rows.

It wasn't getting updated when a scraper db get's cleared, so I've added reindex to that action (https://github.com/openaustralia/morph/commit/3286fcb72407513622b24878fed45bbcd1acfe1f) which does the trip.

There's still the situation you describe where a run changes the number of rows to 0. I can't actually work out a way to test if this is happening. That does seem like a real corner case to me, but what do you think?